addons/web/static/tests/views/kanban/kanban_view_tests.js

```diff
@@ -5060,13 +5060,12 @@ QUnit.module("Views", (hooks) => {
         // state must not be draggable:
         // state is not readonly in the model. state is passed in the arch specifying readonly="1".
         // foo must not be draggable:
-        // foo is readonly in the model fields. foo is passed in the arch but without specifying readonly.
+        // foo is readonly = "True" in the arch which should work same as readonly="1".
         // bar must not be draggable:
         // bar is readonly in the model fields. bar is not passed in the arch.
         // product_id must not be draggable:
         // product_id is readonly in the model fields. product_id is passed in the arch specifying readonly="0",
         // but the readonly in the model takes over.
-        serverData.models.partner.fields.foo.readonly = true;
```

Comment:
please do not change this. This is another scenario that deserves to be tested as well.